internal/expr/parser.go

```diff
@@ -550,3 +581,8 @@ func (p *Parser) parseInputExpression() (*inputPart, bool, error) {
 	cp.restore()
 	return nil, false, nil
 }
+
+// colNum calculates the current column number taking into account line breaks.
+func (p *Parser) colNum() int {
```

Comment:
Could this function go at the top of the file closer to the defn of the parser. The file is currently (very loosely) order in terms of low level functions to high level functions.

Reply:
Done